app/src/main/java/com/ccc/nameapp/scenes/login/main/MainLoginFragment.kt

```diff
@@ -90,7 +92,11 @@ class MainLoginFragment : DaggerFragment(), MainLoginView {
     }
 
     private fun onNextClick() {
-        if (mValidator.isValidEmail(mView.emailOrPhoneEditText.text.toString(), isCheckTempMails = true)) {
+        if (mValidator.isValidEmail(
+                mView.emailOrPhoneEditText.text.toString(),
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/daolq3012/Structure_Android/blob/9cbff12fc6911fd41007439107765dbec2127af9/app/src/main/java/com/ccc/nameapp/scenes/login/main/MainLoginFragment.kt#L96"></span>Unexpected indentation (16) (should be 12)</td>
    </tr>
  </tbody>
</table>